test/api_tester.py

```diff
@@ -45,12 +45,12 @@ class ApiTester(unittest.TestCase):
 
     def setUp(self):
         self.failed_any_test = False
-        self.default_storage = 'Local-CI'
+        self.default_storage = 'Local-CI-Linux'
         self.cloud_test_folder_prefix = 'ImagingCloudTestPython'
         self.original_data_folder = 'ImagingIntegrationTestData'
         self._server_access_file = 'serverAccess.json'
         self._api_version = 'v3.0'
-        self._base_url = 'https://api.aspose.cloud/'
+        self._base_url = 'http://localhost:57972'
```

Comment:
Is it correct URL?

Reply:
Local debug info slipped into the commit. Fixed